test/e2e_node/standalone_test.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/apimachinery/pkg/util/wait"
+	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
+	"k8s.io/kubernetes/pkg/cluster/ports"
+	"k8s.io/kubernetes/test/e2e/framework"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	"github.com/onsi/ginkgo/v2"
+)
+
+var _ = SIGDescribe("[Feature:StandaloneMode] ", func() {
+	f := framework.NewDefaultFramework("static-pod")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
+	ginkgo.It("can create a static Pod ", func(ctx context.Context) {
+
+		var ns, podPath, staticPodName string
+
+		ns = f.Namespace.Name
+		staticPodName = "static-pod-" + string(uuid.NewUUID())
+
+		podPath = framework.TestContext.KubeletConfig.StaticPodPath
+
+		err := createBasicStaticPod(podPath, staticPodName, ns,
+			imageutils.GetE2EImage(imageutils.Nginx), v1.RestartPolicyAlways)
+		framework.ExpectNoError(err)
+
+		file := staticPodPath(podPath, staticPodName, ns)
+		defer os.Remove(file)
+
+		pod := pullPods(ctx, 1*time.Minute, 5*time.Second, staticPodName)
+
+		framework.ExpectEqual(pod.Status.Phase, v1.PodRunning)
```

Comment:
This needs to be wrapped in a gomega.Eventually or else it will fail if it takes some time for the pod to enter running

Reply:
this comment materialized in the job failures =). Let me fix it